torchaudio/datasets/vctk.py

```diff
@@ -26,10 +26,10 @@ def load_vctk_item(fileid: str,
                    folder_audio: str,
                    folder_txt: str,
                    downsample: bool = False) -> Tuple[Tensor, int, str, str, str]:
-    speaker_id, utterance_id = fileid.split("_")
+    speaker_id, utterance_id, _ = fileid.split("_")
 
     # Read text
-    file_txt = os.path.join(path, folder_txt, speaker_id, fileid + ext_txt)
```

Comment:
Slicing string like this is harder to read what is changed. Is there a simple way to do split string? If there is not, can you add a comment?

Reply:
Added appropriate comments